modules/library/render/src/main/java/org/geotools/renderer/style/SLDStyleFactory.java

```diff
@@ -983,6 +987,12 @@ private Geometry findGeometry(final Object feature, String geomName) {
         if(kerning) {
             javaFont = applyKerning(javaFont);
         }
+        double spacing = voParser.getDoubleOption(symbolizer, TextSymbolizer.CHAR_SPACING_KEY, 
+                TextSymbolizer.DEFAULT_CHAR_SPACING);
+        if(spacing != 0) {
```

Comment:
It would, but then kerning would be set and java2d might end up doing more work (depending on its implementation). Since spacing is going to be 0 almost always, I thought it was worth it having the extra if.